README.md

```diff
@@ -243,4 +241,41 @@ On MacOS or Windows systems, we recommend either using a VM or the provided `.de
 - `build`: builds and tags `envbuilder:latest` for your current architecture.
 - `develop`: runs `envbuilder:latest` against a sample Git repository.
 - `test`: run tests.
-- `test-registry`: stands up a local registry for caching images used in tests.
\ No newline at end of file
+- `test-registry`: stands up a local registry for caching images used in tests.
+
+<!--- PLEASE DO NOT MODIFY THIS SECTION. IT IS AUTOMATICALLY GENERATED. --->
+<!--- start-flags --->
+## Flags
+
+| Flag | Env | Default | Description |
+| - | - | - | - |
+| setup-script | SETUP_SCRIPT |  | SetupScript is the script to run before the init script. It runs as the root user regardless of the user specified in the devcontainer.json file. SetupScript is ran as the root user prior to the init script. It is used to configure envbuilder dynamically during the runtime. e.g. specifying whether to start systemd or tiny init for PID 1. |
```

Comment:
```suggestion
| `--setup-script` | `SETUP_SCRIPT` |  | SetupScript is the script to run before the init script. It runs as the root user regardless of the user specified in the devcontainer.json file. SetupScript is ran as the root user prior to the init script. It is used to configure envbuilder dynamically during the runtime. e.g. specifying whether to start systemd or tiny init for PID 1. |
```
Should we prefix flags with `--` and change style to code?

Reply:
From reviewing the existing README and codebase, I don't believe that flag-style configuration is supported by `envbuilder`; only environment variables. This makes sense, given that it is primarily meant to be run inside a container.
Adding a `--` prefix will confuse users who will expect this style of invocation to work.

Reply:
@johnstcn `coder/serpent` supports flags out of the box like env variables so in theory, it should work.